Source/Scene/Primitive.js

```diff
@@ -461,11 +461,18 @@ define([
     });
 
     function cloneAttribute(attribute) {
+        var clonedValues;
+        if(isArray(attribute.values)) {
+            clonedValues = attribute.values.slice(0);
+        }
+        else {
```

Comment:
These kind of fixes can me made automatically if you run the code formatter 
(highlight the code and hit `ctrl+shift+f` in eclipse, i'm not sure what the command is in webstorm)